managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+
+9. When finished, make sure your data is stored in your new database by [connecting to it](/managed-databases/postgresql-and-mysql/how-to/connect-database-instance/), and performing a query.
+
+If the process fails and some data was already partly transferred, we suggest that you activate the **Clean before restore** option in the **Data options** tab to remove partly transferred data.
+
+### Using Airbyte
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Access to [Airbyte Cloud](https://cloud.airbyte.com/) or have [Airbyte installed](https://docs.airbyte.com/deploying-airbyte/quickstart) on your machine.
+
+<Message type="note">
+To avoid any statement timeout errors with Managed Database for PostgreSQL (e.g. `ERROR: canceling statement due to statement timeout`), we recommend you set up the PostgreSQL parameter `statement_timeout` to `1200000` (20 minutes), as recommended by Airbyte for other PostgreSQL-compatible data sources. You can modify this parameter in your [database overview in the Scaleway console](https://console.scaleway.com/rdb/instances), by clicking **More**, then **Advanced Settings**.
+</Message>
+
+#### Adding your existing database as a source
+
+1. From the Airbyte user interface, click **Sources**.
+
+2. Select **Postgres** from the list of sources.
+
+3. Fill in the required information to connect to your existing database:
+    - **Host**
+    - **Port**
+    - **Database Name**
+    - **Username**
+    - **Password**
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+4. Select an SSL Mode and an SSH Tunnel Method.
+
+5. Select **Detect Changes with Xmin System Column** as **Update Method** (other methods are not supported).
+
+4. Click **Set up source**.
+
+#### Adding your new Managed Database for PostgreSQL as a destination
+
+1. Click **Create a connection**.
+
+2. Select **Postgres** from the list of destinations.
+
+3. Fill in the required information to connect to your existing database:
+    - **Host**
+    - **Port**
+    - **Database Name**
+    - **Username**
+    - **Password**
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
```

Comment:
Managed PostgreSQL is not integrated with IAM yet so we need to replace this with "the username & password that you defined upon Database Instance Creation"